### PR TITLE
Fix readable_outputs in CommandResult

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25727
fixes: https://github.com/demisto/etc/issues/25921
## Description
If only readable_outputs supplied, to_context function returned nothing.
fixed.
Failed tests: 
- https://app.circleci.com/pipelines/github/demisto/content/16792/workflows/0439ba2c-0bad-4ac6-9e3c-2f95ac8317a0/jobs/59190
- https://app.circleci.com/pipelines/github/demisto/content/17242/workflows/07a5d017-65ee-4068-903a-492a25afb092/jobs/59716/parallel-runs/1?filterBy=ALL

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py [hunk 1]
```diff
@@ -922,6 +915,11 @@ def test_file_indicators(self):
 
 
 class TestCommandResults:
+    def test_readable_only_context(self):
+        from CommonServerPython import CommandResults
+        context = CommandResults(outputs=None, outputs_prefix='', outputs_key_field='', readable_output='## Something').to_context()
```
Comment (anara123) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:920]:
```suggestion
        context = CommandResults(readable_output='## Something').to_context()
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 2]
```diff
@@ -2581,10 +2581,14 @@ class CommandResults:
     :return: None
     :rtype: ``None``
     """
-    def __init__(self, outputs_prefix, outputs_key_field, outputs, indicators=None, readable_output=None,
+    def __init__(self, outputs_prefix='', outputs_key_field='', outputs=None, indicators=None, readable_output=None,
```
Comment (anara123) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:2584]:
```suggestion
    def __init__(self, outputs_prefix=None, outputs_key_field=None, outputs=None, indicators=None, readable_output=None,
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 3]
```diff
@@ -2619,9 +2626,6 @@ def to_context(self):
             else:
                 human_readable = self.readable_output
```
Comment (anara123) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:2627]: you can delete these lines

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py (hunk 2 above)
Comment (jochman) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:2584]: string isn't a mutable object, so it's fine to use empty strings.

**Comment by Itay4** [comment]
@roysagi please review rn
@jochman please resolve conflicts

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py [hunk 4]
```diff
@@ -922,6 +915,42 @@ def test_file_indicators(self):
...
+
+        When:
+        - Returning results
+
+        Then:
+        - Validate HumanReadable exists
+        """
+        from CommonServerPython import CommandResults
+        context = CommandResults(readable_output='## Something').to_context()
+        assert context.get('HumanReadable')
```
Comment (anara123) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:931]:
```suggestion
        assert context.get('HumanReadable') == '## Something'
```

**Pull request merged**